### ISSUE TITLE
sar-c reader swath identifier 

### ISSUE THREAD

**Issue by ptcongbk** [pytroll/satpy#1735]
In sar-c_safe.yaml, it is defined the pattern as 
`'{fmission_id:3s}_{fsar_mode:2s}_{fproduct_type:3s}{fresolution:1s}_{fprocessing_level:1s}{fproduct_class:1s}{fpolarization:2s}_{fstart_time:%Y%m%dT%H%M%S}_{fend_time:%Y%m%dT%H%M%S}_{forbit_number:6d}_{fmission_data_take_id:6s}_{fproduct_unique_id:4s}.SAFE/measurement/{mission_id:3s}-**{swath_id:2s}**-{product_type:3s}-{polarization:2s}-{start_time:%Y%m%dt%H%M%S}-{end_time:%Y%m%dt%H%M%S}-{orbit_number:6d}-{mission_data_take_id:6s}-{image_number:3s}.tiff'`

But based on the level-1 product formatting (https://sentinel.esa.int/web/sentinel/technical-guides/sentinel-1-sar/products-algorithms/level-1-product-formatting), the swath identifier can be 2 or 3 characters which made it impossible to read.

Am I wrong or we should modify the yaml file in this case?

**Comment by mraspaud** [comment]
@ptcongbk good catch, thanks for reporting this! You are absolutely right, the pattern now forces only 2 characters, so it should be fixed. Do you think you can open a PR for this?

**Comment by ptcongbk** [comment]
@mraspaud many thanks for your clarification. Could you please help me with this PR since I am not quite sure what is the best way to fix it?
